### PR TITLE
Feat: Add tips section

### PR DESCRIPTION
**Pull request by dipinknair**
- 3D visualization
- export image
- project tree
- app_libraries

### CODE REVIEW

**Comment by dipinknair** [comment]
image

File: examples/00_tips/tips_01.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+# %%
+# Embed mechanical and set global variables
+
+app = mech.App(version=242)
+app.update_globals(globals())
+print(app)
+
+
+# %%
+# Download geometry and import
```
Comment (klmcadams) [inline examples/00_tips/tips_01.py:26]:
```suggestion
# Download and import geometry
```

File: examples/00_tips/tips_02.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+# %%
+# Embed mechanical and set global variables
+
+app = mech.App(version=242)
+app.update_globals(globals())
+print(app)
+
+
+# %%
+# Download geometry and import
```
Comment (klmcadams) [inline examples/00_tips/tips_02.py:27]:
```suggestion
# Download and import geometry
```

File: examples/00_tips/tips_02.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+settings_720p.Width = 1280
+settings_720p.Height = 720
+settings_720p.CurrentGraphicsDisplay = False
+
+# Rotate the geometry if needed
+ExtAPI.Graphics.Camera.Rotate(180, CameraAxisType.ScreenY)
+
+
+# %%
+# custom function for displaying the image
```
Comment (klmcadams) [inline examples/00_tips/tips_02.py:62]:
```suggestion
# Custom function for displaying the image
```

File: examples/00_tips/tips_02.py [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+from matplotlib import image as mpimg
+from matplotlib import pyplot as plt
+
+# Temporary directory to save image
```
Comment (klmcadams) [inline examples/00_tips/tips_02.py:68]:
```suggestion
# Temporary directory to save the image
```

File: examples/00_tips/tips_02.py [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+    plt.figure(figsize=(16, 9))
+    plt.imshow(mpimg.imread(os.path.join(cwd, image_name)))
+    plt.xticks([])
+    plt.yticks([])
+    plt.axis("off")
+    plt.show()
+
+
+# %%
+# Export Image and display
```
Comment (klmcadams) [inline examples/00_tips/tips_02.py:82]:
```suggestion
# Export and display the image
```

File: examples/00_tips/tips_02.py [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+
+# Fits the geometry in viewing area
```
Comment (klmcadams) [inline examples/00_tips/tips_02.py:85]:
```suggestion
# Fits the geometry in the viewing area
```

File: examples/00_tips/tips_02.py [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+Graphics.Camera.SetFit()
+
+Graphics.ExportImage(
+    os.path.join(cwd, "geometry.png"), image_export_format, settings_720p
+)
+
+# Display image using matplotlib
```
Comment (klmcadams) [inline examples/00_tips/tips_02.py:92]:
```suggestion
# Display the image using matplotlib
```

File: examples/00_tips/tips_03.py [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+# Embed mechanical and set global variables
+
+app = mech.App(version=242)
+app.update_globals(globals())
+print(app)
+
+
+# %%
+# Download mechdb file
+# ~~~~~~~~~~~~~~~~~~~~
```
Comment (klmcadams) [inline examples/00_tips/tips_03.py:27]:
```suggestion
# Download the mechdb file
# ~~~~~~~~~~~~~~~~~~~~~~~~
```

File: examples/00_tips/tips_03.py [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+
+mechdb_path = download_file("graphics_test.mechdb", "pymechanical", "test_files")
+
+# %%
+# Load the mechdb inside Mechanical
```
Comment (klmcadams) [inline examples/00_tips/tips_03.py:32]:
```suggestion
# Load the mechdb file inside Mechanical
```

File: examples/00_tips/tips_03.py [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+app.open(mechdb_path)
+
+# %%
+# Display the project tree
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+
+app.print_tree()
+
+# %%
+# Display tree only under first analysis
```
Comment (klmcadams) [inline examples/00_tips/tips_03.py:43]:
```suggestion
# Display the tree only under the first analysis
```

File: examples/00_tips/tips_02.py [hunk 11]
```diff
@@ -0,0 +1,100 @@
...
+# Import necessary libraries
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+import os
+
+import ansys.mechanical.core as mech
+from ansys.mechanical.core.examples import delete_downloads, download_file
+
+# %%
+# Embed mechanical and set global variables
```
Comment (klmcadams) [inline examples/00_tips/tips_02.py:19]:
```suggestion
# Embed Mechanical and set global variables
```

File: examples/00_tips/tips_03.py [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+# %%
+# Import necessary libraries
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+import ansys.mechanical.core as mech
+from ansys.mechanical.core.examples import delete_downloads, download_file
+
+# %%
+# Embed mechanical and set global variables
```
Comment (klmcadams) [inline examples/00_tips/tips_03.py:18]:
```suggestion
# Embed Mechanical and set global variables
```

**Review by klmcadams — APPROVED** [review]

**Pull request merged**